### ISSUE TITLE
[AdaptiveCards- Designer - Target Version]: Screen Reader does not convey the 'warning and validation' message when user is selecting the latest version.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8173]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: [https://adaptivecards.io/](https://nam06.safelinks.protection.outlook.com/?url=https://adaptivecards.io/&data=04%7c01%7cv-ibalam%40microsoft.com%7cbd35d875ec3f4539762308d88f709775%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637417061036711036%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=dUolQbKGi1J8gsoxC2JG5TGoNi41msPfIIjKSIyPGHM%3D&reserved=0)
OS Version: 22H2(OS Build 25247.1000)
Browser Version: Version 110.0.1543.0 (Official build) dev (64-bit)
Screen Reader: Narrator

**Repro Steps:**
1. Turn on Narrator by Ctrl+Win+Enter keys.
2. Open above URL with valid credential.
3. Navigate to 'Designer' link and press enter key to activate it.
4. Navigate to 'Redo' button and press tab key.
5. Navigate to 'Target Version' button and press enter key to activate it.
6. Navigate to the '1.5' menu item and press enter key and observe the issue.

**Actual Result:**
Screen Reader does not convey the warning and validation message which appears when user is selecting the '1.5' version from target version combo box.

**Expected Result:**
Screen Reader should announce the warning and validation message to user after selecting the '1.5' target version from the combo box.

**Note:**
1. Same issue is repro when user is selecting the Window timeline menu items.
2. Same issue repro throughout the Designer page for the error message.
3. Same issue is repro with NVDA as well.

**User Experience:**
It will impact the screen reader user if the screen reader does not convey the message to the user.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7B9C514377-76E5-4CFE-8300-DA3079D3CDC3%7D&file=MAS%201.3.1%20%E2%80%93%20Info%20and%20Relationships.docx&action=default&mobileredirect=true&share=IQF3Q1Gc5Xb-TIMA2jB5083DAcWOZyaXcx7TtNeRlXM4uHk

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

https://user-images.githubusercontent.com/95913748/208673410-1d1e86a7-db66-4a40-bc0f-a012bce4953a.mp4

### Card JSON

```JSON
NA
```
